crates/nu-source/src/meta.rs

```diff
@@ -280,6 +289,7 @@ impl From<(usize, usize, AnchorLocation)> for Tag {
         Tag {
             anchor: Some(anchor),
             span: Span::new(start, end),
+            data_color: None,
```

Comment:
All of the data_colors assigned None here in `meta.rs` should be reviewed, some of these should be values that are passed in.